.github/workflows/build.yml

```diff
@@ -54,4 +54,30 @@ jobs:
           mvn clean verify -pl tcks/microprofile-lra -Dtcks -Dlra.coordinator.url=http://localhost:8080/lra-coordinator
           kill %1
 
+  test-containers:
+    runs-on: ubuntu-latest
+    name: Integration verification with Quarkus and test-containers
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: 21
+          distribution: temurin
+          cache: maven
+
+      - name: Build with Maven
+        run: mvn -B install
 
+      - name: Checkout Quarkus
+        uses: actions/checkout@v2
+        with:
+          repository: quarkusio/quarkus
+          path: ./quarkus
+
+      - name: Quarkus integration
+        working-directory: ./quarkus
+        run: |
+          mvn -Dquickly
+          mvn clean install -pl tcks/microprofile-lra -Dtcks -Dstart-containers -Dtest-containers
```

Comment:
This will not test the proposed build but the latest published image. You need to build the image here manually and test it.

Reply:
You are right, I will update this PR.